### PR TITLE
Fixes after CR

### PR DESCRIPTION
**Pull request by kvas-damian**
Additionally:
Distance comparision method changed to more precise one without performance degradation.

### CODE REVIEW

File: js-delayed-hover.js [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		/**
 		 * Just simple one-time-defined function that will do nothing
 		 */
-		noop: function () {},
+		noop: function() {},
```
Comment (hakubo) [inline js-delayed-hover.js]: you can use `Function.prototype` instead of noop

File: js-delayed-hover.js [hunk 2]
```diff
@@ -5,19 +5,19 @@
...
- *				onDeactivate: function() { // Function to call when you stop hovering element
- *					container.classList.remove( 'active' );
- *				}
+ *	delayedHover(
+ *		container, // HTML node element to watch
+ *		{
+ *			checkInterval: 100, // How often mouse speed is calculated
+ *			maxActivationDistance: 20, // Breakpoint for activation, onActivate callback will be fired when mouse speed is below this value
+ *			onActivate: function () { // Function to call when mouse speed meets required conditions
+ *				container.classList.add( 'active' );
```
Comment (hakubo) [inline js-delayed-hover.js:14]: wow the dooms day
that mobile web team have to use older API than desktop!
as we have to support A2.3 that don't have classList...

Comment (kvas-damian) [inline js-delayed-hover.js:14]: I'm so sorry.... Not, finally this time has come ;)

Comment (nandy-andy) [inline js-delayed-hover.js:14]: :crying_cat_face:

**Pull request merged**